test/components/views/dialogs/ForwardDialog-test.tsx

```diff
@@ -193,21 +186,22 @@ describe("ForwardDialog", () => {
             event: true,
         });
 
-        const wrapper = await mountForwardDialog(replyMessage);
-        expect(wrapper.find("ReplyChain")).toBeTruthy();
+        await mountForwardDialog(replyMessage);
+
+        expect(screen.queryByText("Hi Alice!", { exact: false })).toBeInTheDocument();
     });
 
     it("disables buttons for rooms without send permissions", async () => {
         const readOnlyRoom = mkStubRoom("a", "a", mockClient);
         readOnlyRoom.maySendMessage = jest.fn().mockReturnValue(false);
         const rooms = [readOnlyRoom, mkStubRoom("b", "b", mockClient)];
 
-        const wrapper = await mountForwardDialog(undefined, rooms);
+        const { container } = await mountForwardDialog(undefined, rooms);
+
+        const [firstButton, secondButton] = container.querySelectorAll<HTMLButtonElement>(".mx_ForwardList_sendButton");
 
-        const firstButton = wrapper.find("AccessibleButton.mx_ForwardList_sendButton").first();
-        expect(firstButton.prop("disabled")).toBe(true);
-        const secondButton = wrapper.find("AccessibleButton.mx_ForwardList_sendButton").last();
-        expect(secondButton.prop("disabled")).toBe(false);
+        expect(firstButton.getAttribute("aria-disabled")).toBeTruthy();
+        expect(secondButton.getAttribute("aria-disabled")).toBeFalsy();
```

Comment:
We seem to have lost the actual `disabled` checks? Those drive functionality, aria-disabled drives accessibility. I think there is a `toBeDisabled()` jest matcher added by RTL

Reply:
`firstButton` and `secondButton` are `div`s with `role=button`. The `disabled` attribute is not allowed on `div`s... so the `aria-disabled` is the right way to check here.

`disabled={true}` in JSX translates to `disabled=""` in HTML when used on a `div` and to `disabled="disabled"` when used on an `input`. And the `toBeDisabled()` helper does not work in this context.